### PR TITLE
Add `edrixs` package

### PR DESCRIPTION
**Pull request by mrakitin**
This is another attempt to package https://github.com/NSLS-II/edrixs. The first one was via the currently closed https://github.com/conda-forge/staged-recipes/pull/15729.

---

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mrakitin** [comment]
@conda-forge/staged-recipes, could you please help forcing the testing stage to install the MPI/openmpi variant of `arpack` in the test environment? Currently, it is setting it to [`arpack:           3.8.0-nompi_h0baa96a_100   conda-forge`](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=568718&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2395) no matter how I pin it, which causes failures.

The MPI variant of `arpack` was built via https://github.com/conda-forge/arpack-feedstock/pull/46. I wonder if I missed any `run_exports` ([docs](https://conda-forge.org/docs/maintainer/knowledge_base.html#message-passing-interface-mpi)). Any ideas? Thanks!

**Comment by mrakitin** [comment]
I guess https://github.com/conda-forge/arpack-feedstock/pull/47 may fix the problem, but I am not sure if it's OK to mix two styles of exports (one with `pin_subpackage` and another one with the variant). I wanted to confirm it first with you.

@beckermr, thanks for merging the previous PR for `arpack` earlier today. Could you please check if the new one there makes sense?

**Comment by mrakitin** [comment]
Thanks for your help, @beckermr! I think it's ready to go.

@conda-forge/staged-recipes, ready for your review!

**Comment by mpmdean** [comment]
I agree to help co-maintain this.

File: recipes/edrixs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  sha256: fce7c1543d61df156696533dd7abcb7782e410bfb91afe7ae77109735146d160
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - make
+    - openblas
```
Comment (beckermr) [inline recipes/edrixs/meta.yaml:19]: Why is openblas in build? This should not be needed.

Comment (mrakitin) [reply recipes/edrixs/meta.yaml]: Good catch! Probably it's a leftover from the previous attempts to make it work and to address missing libraries. I committed a fix, let's see if it works without it.

**Comment by mrakitin** [comment]
@carterbox, thanks for chiming in. What is the `python-c` label for? The project has [Fortran bindings](https://github.com/NSLS-II/edrixs/blob/ef33ca6769aefdb72c5031ab518098dea8a1628e/setup.py#L48-L50), no C/C++.

File: recipes/edrixs/meta.yaml (hunk 1 above)
Comment (mrakitin) [reply recipes/edrixs/meta.yaml]: Done, the build is successful. Could you please re-review it?

**Comment by carterbox** [comment]
We don't have a `python-fortran` team, so the `python-c` language team is the most appropriate.

**Review by beckermr — APPROVED** [review]

**Comment by mrakitin** [comment]
Awesome, thank you for the quick turnaround and for your help with the dependencies!

**Pull request merged**